frontend/src/components/table_view.tsx

```
@@ -0,0 +1,132 @@
+        {path
+          ? path.map((val, i) => {
+              const end_char = i !== path.length - 1 ? "/" : "";
+
+              const name = val[0];
+              const id = val[1];
+
+              return (
+                <span
+                  onClick={(e) => set_breadcrumb_id(e, id)}
+                  className="breadcrumb"
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)